### PR TITLE
added a generic brb topic

### PR DESCRIPTION
**Pull request by multimokia**
#3745 #607 #3526 #2549 #2033 #497
- adding issues

Adds a generic brb idle transition.

### CODE REVIEW

**Comment by ThePotatoGuy** [comment]
need to add some stuff

File: Monika After Story/game/script-topics.rpy [hunk 1]
```diff
@@ -9349,6 +9349,63 @@ label monika_idle_game_quick_callback:
...
+            prompt="I'll be right back",
+            category=['be right back'],
+            pool=True,
+            unlocked=True
+        )
+    )
+
+label monika_brb_idle:
+    if mas_isMoniAff(higher=True):
+        m 1eua "Alright [player]."
```
Comment (jmwall24) [inline Monika After Story/game/script-topics.rpy]:
```suggestion
        m 1eua "Alright, [player]."
```

File: Monika After Story/game/script-topics.rpy [hunk 2]
```diff
@@ -9349,6 +9349,63 @@ label monika_idle_game_quick_callback:
...
+    #Set up the callback label
+    $ mas_idle_mailbox.send_idle_cb("monika_brb_idle_callback")
+    #Then the idle data
+    $ persistent._mas_idle_data["monika_idle_brb"] = True
+    return "idle"
+
+label monika_brb_idle_callback:
+    python:
+        wb_quips = [
+            "So what else did you want to do today?",
```
Comment (jmwall24) [inline Monika After Story/game/script-topics.rpy]:
```suggestion
            "So, what else did you want to do today?",
```

**Comment by Ulvsblakk** [comment]
Love this!

**Pull request merged**